images/kubernetes-csi-external-provisioner/configs/latest.apko.yaml

```diff
@@ -0,0 +1,31 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - kubernetes-csi-external-provisioner
+    - wolfi-baselayout
+
+accounts:
+  groups:
```

Comment:
are `groups/users` needed at all if we're just going to use `root`?

Reply:
yeah I wasn't sure about that either, @imjasonh WDYT?